services/scan/src/backup.ts

```diff
@@ -85,24 +85,30 @@ export class Backup {
       throw new Error('cannot backup without election configuration');
     }
 
+    debug('adding election.json to backup...');
     await this.addEntry('election.json', electionDefinition.electionData);
+    debug('added election.json to backup');
 
-    const cvrStream = new WritableStream();
-    await this.store.exportCvrs(cvrStream);
-    await this.addEntry('cvrs.jsonl', cvrStream.toBuffer());
+    debug('adding CVRs to backup...');
+    const cvrStream = new PassThrough({ highWaterMark: 100000000 }); // 100MB
```

Comment:
I tested this without the `highWaterMark` option and it worked, at least on a small backup.

Reply:
If we keep this, we can at least use the thousands separator:
```suggestion
    const cvrStream = new PassThrough({ highWaterMark: 100_000_000 }); // 100MB
```